### ISSUE TITLE
VoiceOver does not announce element with autoFocus in MAC

### ISSUE THREAD

**Issue by Vinit-Pandit** [adobe/react-spectrum#9821]
### Provide a general summary of the issue here

When using the autoFocus prop, VoiceOver does not update focus correctly, so the focused element is not read.

### 🤔 Expected Behavior?

focused element should get read in voiceover

### 😯 Current Behavior

When using the autoFocus prop, VoiceOver does not update focus correctly, so the focused element is not read

### 💁 Possible Solution

Adding delay before focusing with `autofocus`

### 🖥️ Steps to Reproduce

https://github.com/user-attachments/assets/a8cc33d3-fc4f-42a3-ad8f-2f9a9c00f270

### Version

Latest

### What browsers are you seeing the problem on?

Safari

### What operating system are you using?

MAC

**Comment by snowystinger** [comment]
Closing as a duplicate of https://github.com/adobe/react-spectrum/issues/9814#issuecomment-4097049480
